CONTRIBUTING.md

```diff
@@ -7,9 +7,16 @@ CONTRIBUTING TO VGSTATION
  * This means, primarily, that you shouldn't fix bugs **and** add content in the same PR. When we mean 'bundling', we mean making one PR for multiple, unrelated changes.
 * **Test your changes.**  PRs that do not compile will not be accepted.
  * Testing your changes locally is incredibly important. If you break the serb we will be very upset with you.
-* **Large changes require discussion.**  If you're doing a large, game-changing modification, or a new layout for something, discussion with the community is required as of 26/6/2014.  Map and sprite changes require pictures of before and after.  **MAINTAINERS ARE NOT IMMUNE TO THIS.  GET YOUR ASS IN IRC.**
+* **Large changes require discussion.**  If you're doing a large, game-changing modification, or a new layout for something, discussion with the community is required as of 26/6/2014.  Map and sprite changes require pictures of before and after.  **MAINTAINERS ARE NOT IMMUNE TO THIS.  GET YOUR ASS IN THE CODE DISCORD.** (link in README.md)
```

Comment:
I think the minimum discussion duration is already handled by the 24h before merge portion.